db_parser.py

```diff
@@ -15,76 +15,88 @@ def parse_db(filename):
 
     """
 
-    # This regex matches the EPICS convention for PV names, e.g. a-z A-Z 0-9 _ - : [ ] < > ;
-    # The ISIS convention uses a reduced set (a-z A-Z 0-9 _ :) but this is not relevant at this level
-    reg_record_start = 'record\s*\((\w+),\s*"([\w_\-\:\[\]<>;$\(\)]+)"\)'
+    # This regex matches the EPICS convention for PV names,
+    # e.g. a-z A-Z 0-9 _ - : [ ] < > ;
+    # The ISIS convention uses a reduced set (a-z A-Z 0-9 _ :)
+    # but this is not relevant at this level
+    reg_record_start = r'record\s*\((\w+),\s*"([\w_\-\:\[\]<>;$\(\)]+)"\)'
     # Aliases look like alias("RECORDNAME","ALIASNAME")
-    reg_alias_alone = 'alias\s*\(\s*"([\w_\-\:\[\]<>;$\(\)]+)",\s*"([\w_\-\:\[\]<>;$\(\)]+)"\)'
+    reg_alias_alone = r'alias\s*\(\s*"([\w_\-\:\[\]<>;$\(\)]+)",' \
+                      r'\s*"([\w_\-\:\[\]<>;$\(\)]+)"\)'
```

Comment:
Don't split regexes over multiple lines - even if it goes over the style convention's limit readability wins over convention here...